### ISSUE TITLE
Is the library API stable? 

### ISSUE THREAD

**Issue by jackzhangdev** [alibaba/testable-mock#159]
Is the library API stable? Can it be used in production?

**Comment by linfan** [comment]
No actually, some incompatible changes are still expected before the final `v1.0` release, but [upgrade guide](https://alibaba.github.io/testable-mock/#/en-us/doc/upgrade-guide) will always be available between versions, and it won't be hard to follow up.

As we know, there are already several large projects (and also many more smaller ones) are using this library for real, and the latest version (currently `v0.6.5`) could be consider as "stable enough". However, if anything goes wrong, just raise an issue, we'll look into it asap.

**Comment by jackzhangdev** [comment]
thanks
